src/TabsOnEdit.php

```diff
@@ -177,4 +161,31 @@ private function resolveResource(NovaRequest $request)
 
         return $request->newResource();
     }
+
+    private function assignTabPanels(FieldCollection $fields) 
+    {
+        $non_tab_fields = $tab_panels = [];
+
+        $fields->each(function($field, $key) use (&$non_tab_fields, &$tab_panels) {
+            if ( isset($field->meta['tab']) ) {
+                if ( isset($tab_panels[$field->panel]) ) {
+                    $tab_panels[$field->panel]['fields'][] = $field;
+                } else {
+                    $new_panel = [
+                        'component' => 'tabs',
+                        'panel' => $field->panel,
+                        'fields' => [$field]
+                    ];
+                    $tab_panels[$field->panel] = $new_panel;
+                }
+            } else {
+                $non_tab_fields[] = $field;
+            }
+        });
+
+        return new TabsFieldCollection([
+            ...$non_tab_fields,
```

Comment:
As this package still targets PHP 7.1, please use `array_merge` instead.

Using this would require a major bump.